nilearn/surface/surface.py

```diff
@@ -1237,3 +1237,50 @@ def check_surface(surface):
     surface = load_surface(surface)
     mesh, data = check_mesh_and_data(surface.mesh, surface.data)
     return Surface(mesh, data)
+
+
+def mesh_to_gifti(
+    coordinates,
+    faces,
+    gifti_file,
+):
+    """Write surface mesh to gifti file on disk."""
```

Comment:
If this is meant to be part of the public API, the docstring should be more complete

Reply:
yeah
I was thinking of doing a global pass to update them all at the end but probably better to do it as we integrate the new code.

Reply:
should be better now